### ISSUE TITLE
Device Identification failed on firefox 67 beta

### ISSUE THREAD

**Issue by seringue** [CaliOpen/Caliopen#1379]
Device Identification failed on firefox 67 beta

1. Each time I login it tells me that it is a new device, send a mail to verify it. 
2. From thunderbird, I click the link in the mail which opens it in a new tab, asks for login then verify the device.
3.  close the new tab, go back to the pinned one, after a while it asks for login again and then back to step 1.

I don't know if it's a good idea on a security level but if the cookies from previous authorized sessions is there may be it's an authorized device?

![cal01](https://user-images.githubusercontent.com/11724116/58080559-f3665880-7ba2-11e9-9d99-8b40631b92a3.jpg)

**Comment by MonsieurLanza** [comment]
The device is identified from data set in localStorage. 
If firefox is configured to discard local data after each session, Caliopen cannot authentify it next time, as it will create a new device ID & signature key pair.

**Comment by seringue** [comment]
I don't think this is the case here, I have deactivated anti-tracking for Caliopen in firefox and allowed cookies and local storage in Cookie Autodelete extension. Moreover, even without closing firefox, just disconnect reconnect does it. 


![cal02](https://user-images.githubusercontent.com/11724116/58094508-66ca9300-7bc0-11e9-9bbb-f94d1ae77203.jpg)

**Comment by seringue** [comment]
My bad, thought deactivating blocking was also allowing cookies, turns out I also needed a separate authorization for cookies...
